### PR TITLE
change yverdon to mapnv

### PR DESCRIPTION
**Pull request by monodo**
Little relooking @kalbermattenm

### CODE REVIEW

File: content/pages/1_community.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The UserGroup is composed of representatives of the following entities:
 * City of Nyon
 * City of Prilly
 * City of Pully
-* City of Yverdon-les-Bains
+* Géoportail du Nord Vaudois
```
Comment (rbovard) [inline content/pages/1_community.md]: Please reorder entry alphabetically

File: content/pages/1_community_de.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Die Benutzergruppe besteht aus Vertreter von folgenden Entitäten:
 * Stadt Nyon
 * Stadt Prilly
 * Stadt Pully
-* Stadt Yverdon-les-Bains
+* Géoportail du Nord Vaudois
```
Comment (rbovard) [inline content/pages/1_community_de.md]: Same here

File: content/pages/1_community_fr.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Le UserGroup est composé de représentants des entités suivantes :
 * Ville de Nyon
 * Ville de Prilly
 * Ville de Pully
-* Ville d'Yverdon-les-Bains
+* Géoportail du Nord Vaudois
```
Comment (rbovard) [inline content/pages/1_community_fr.md]: Same here

**Review by rbovard — CHANGES_REQUESTED** [review]
Thanks @monodo just a little sort thing, otherwise LGTM

**Comment by monodo** [comment]
Fixed @rbovard

**Review by rbovard — APPROVED** [review]

**Comment by rbovard** [comment]
Thanks @monodo 

I'll let @kalbermattenm merge if he's happy with it :)

**Comment by kalbermattenm** [comment]
I am always happy when you guys do good work !

Thanks a lot for having reviewed that @rbovard ! Feel free to merge !

**Pull request merged**